pkg/config/template.go

```diff
@@ -224,6 +224,8 @@ type configuration struct {
 		LakefsBaseURL      string        `mapstructure:"lakefs_base_url"`
 	}
 	UI struct {
+		// Enable serving of embedded UI
```

Comment:
Just becase code doc in Go requires to start with the name of the field/func
```suggestion
		// Enabled - control serving of embedded UI
```

Reply:
Fixed